### PR TITLE
Added new navbar with test and removed userMenu

### PR DESCRIPTION
**Pull request by stepek**

### CODE REVIEW

File: src/app/components/Navbar.jsx [hunk 1]
```diff
@@ -0,0 +1,65 @@
...
+        {this.props.tenants.map(item => (
+          <MenuItem key={item.id} text={item.name}
+            onClick={() => this.handleTenantClick(item.id)}
+          />
+        ))}
+      </Menu>
+    );
+
+    return (
+      <nav className="pt-navbar .modifier">
```
Comment (sealionkat) [inline src/app/components/Navbar.jsx]: What is `.modifier'?

File: src/app/App.jsx [hunk 2]
```diff
@@ -14,11 +16,6 @@ const contentStyle = {
   paddingRight: 14
 };
```
Comment (sealionkat) [inline src/app/App.jsx]: Please move inline styles to main.scss :smiling_imp:

Comment (stepek) [inline src/app/App.jsx]: These changes do not concern layout in App.jsx

File: src/app/components/Navbar.jsx (hunk 1 above)
Comment (stepek) [inline src/app/components/Navbar.jsx]: fixed

**Pull request merged**